keyboards/ymdk/id75/chconf.h

```diff
@@ -0,0 +1,40 @@
+/* Copyright 2020 QMK
```

Comment:
This file should be removed

Reply:
compiled and flashed without: no noticable difference, i confirm